perun/utils/structs/common_structs.py

```diff
@@ -677,6 +677,36 @@ def __exit__(self, exc_type: str, exc_val: Exception, exc_tb: traceback.StackSum
         return isinstance(exc_val, self.handler_exc)
 
 
+class SortOrder(Enum):
+    """Enumeration representing sorting order in a more descriptive way compared to the library."""
+
+    Ascending = "ascending"
+    Descending = "descending"
```

Comment:
I seriously can't be bothered to write ascending/descending. I would suggest to shorten this to `asc` and `desc`; it's well-established.